.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,166 @@
+name: Bug Report
+description: Broken or unintended behavior with one of our libraries.
+title: '[Bug] '
+labels: [bug-unconfirmed, question]
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Before Submitting your Issue
+      Please make sure that your question or issue is not already covered in [MSAL docs](https://learn.microsoft.com/en-us/entra/msal/dotnet/) or [samples](https://learn.microsoft.com/en-us/azure/active-directory/develop/sample-v2-code?tabs=apptype).
```

Comment:
Is there any sort of 'common issues and solutions' page to link to? If so you should link it here, and if there isn't a page like that then sending them to the MSAL .NET/sample overview pages probably won't help them anyway.

Reply:
There are troubleshooting sections in specific feature docs but there is no common FAQ page.

Reply:
```suggestion
      Please make sure that your question or issue is not already covered in [MSAL docs](https://learn.microsoft.com/entra/msal/dotnet/) or [samples](https://learn.microsoft.com/azure/active-directory/develop/sample-v2-code?tabs=apptype).
```